GETTING_STARTED.md

```diff
@@ -1154,6 +1154,12 @@ factory :user do
     gender { "Female" }
     login { "#{name} (F)" }
   end
+  
+  trait :other do
+    name { "It Doe" }
```

Comment:
We'd need to use a different name here. "It" might well be somebody's name, but in this particular context it comes across as describing a person with the English word "it". That's not OK.